src/EventSubscriber/RequestSubscriber.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\EventSubscriber;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+
+class RequestSubscriber implements EventSubscriberInterface
+{
+    private $environment;
+    
+    public function __construct(string $environment)
+    {
+        $this->environment = $environment;
+    }
+
+    public static function getSubscribedEvents()
+    {
+        // return the subscribed events, their methods and priorities
+        return [
+            KernelEvents::REQUEST => [
+                ['setupTideways', 10000],
+                ['forceTls', 1000],
+            ],
+            KernelEvents::RESPONSE => [
+                ['setHsts', 1000],
+            ],
+        ];
+    }
+
+    public function setupTideways(RequestEvent $event)
+    {
+        if (!$event->isMasterRequest() || !class_exists('Tideways\Profiler')) {
+            return;
+        }
+    
+        $req = $event->getRequest();
+        $actionName = $req->get('_route');
+        if (strpos($actionName, '__') === 0) {
+            $actionName = $req->get('_controller');
+        }
+        \Tideways\Profiler::setTransactionName($req->getMethod().' '.$actionName);
```

Comment:
tideways has built-in support for Symfony, so this might not be needed

Reply:
Ah true, will remove this thanks